integration-tests/reliable-mqtt/test-executor/src/lib.rs

```diff
@@ -0,0 +1,216 @@
+#[macro_use]
+extern crate log;
+
+#[macro_use]
+extern crate anyhow;
+
+extern crate ctor;
+
+use anyhow::*;
+use async_std;
+use async_std::future::timeout;
+use async_std::prelude::*;
+use async_std::stream;
+use async_std::stream::Stream;
+use async_std::stream::StreamExt;
+use chrono::{DateTime, Utc};
+use cloudevents::{AttributesReader, Data, Event, EventBuilder, EventBuilderV10};
+use env_logger::Env;
+use futures::try_join;
+use paho_mqtt as mqtt;
+use serde_json;
+use std::collections::HashSet;
+use std::env;
+use std::time::Duration;
+
+fn get_event_id(payload: &[u8]) -> Result<String> {
+    let event: Event = serde_json::from_slice(payload)?;
+    Ok(event.id().to_string())
+}
+
+async fn setup_connection(client: mqtt::AsyncClient) -> Result<()> {
```

Comment:
mark the functions that are only used in the test as `#[allow(dead_code)]` to prevent `warning: function is never used: `setup_connection`